backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -190,33 +191,41 @@ export class UserResolver {
     @Ctx() context: any,
   ): Promise<User> {
     email = email.trim().toLowerCase()
-    const result = await apiPost(CONFIG.LOGIN_API_URL + 'unsecureLogin', { email, password })
+    // const result = await apiPost(CONFIG.LOGIN_API_URL + 'unsecureLogin', { email, password })
+    // UnsecureLogin
+    const loginUserRepository = getCustomRepository(LoginUserRepository)
+    const loginUser = await loginUserRepository.findByEmail(email)
+    if (!loginUser) {
+      throw new Error('No user with this credentials')
+    }
```

Comment:
The implementation of `findByEmail` will always find a loginUser or throws an exception. Use catch

Reply:
Thanks for the hint didn't see this while coding!